src/app/containers/SignupContainer/VerificationStep/VerificationForm/VerificationMethodForm/VerificationMethodForm.js

```diff
@@ -30,39 +30,43 @@ const VerificationMethodForm = ({ defaultEmail, allowedMethods, onSubmit }) => {
-                <Row>
-                    <Label>
-                        {isMethodAllowed(VERIFICATION_METHOD.EMAIL) ? (
-                            <div className="mb1">
-                                <Radio
-                                    className="mb1 block"
-                                    checked={method === VERIFICATION_METHOD.EMAIL}
-                                    onChange={handleSelectMethod(VERIFICATION_METHOD.EMAIL)}
-                                >{c('Option').t`Email address`}</Radio>
+                <>
+                    <Row>
+                        <Label>{c('Label').t`Verification method`}</Label>
+                        <Field>
+                            <div className="pt0-5">
+                                {isMethodAllowed(VERIFICATION_METHOD.EMAIL) ? (
+                                    <Radio
+                                        className="mr1"
+                                        checked={method === VERIFICATION_METHOD.EMAIL}
+                                        onChange={handleSelectMethod(VERIFICATION_METHOD.EMAIL)}
+                                    >{c('Option').t`Email address`}</Radio>
+                                ) : null}
+                                {isMethodAllowed(VERIFICATION_METHOD.SMS) && (
+                                    <Radio
+                                        checked={method === VERIFICATION_METHOD.SMS}
+                                        onChange={handleSelectMethod(VERIFICATION_METHOD.SMS)}
+                                    >{c('Option').t`SMS`}</Radio>
+                                )}
                             </div>
-                        ) : null}
-                        {isMethodAllowed(VERIFICATION_METHOD.SMS) && (
-                            <div className="mb1">
-                                <Radio
-                                    checked={method === VERIFICATION_METHOD.SMS}
-                                    onChange={handleSelectMethod(VERIFICATION_METHOD.SMS)}
-                                >{c('Option').t`SMS`}</Radio>
-                            </div>
-                        )}
-                    </Label>
-                    <Field>
-                        {method === VERIFICATION_METHOD.EMAIL && (
-                            <VerificationEmailInput
-                                loading={loading}
-                                defaultEmail={defaultEmail}
-                                onSendClick={handleSendEmailCode}
-                            />
-                        )}
-                        {method === VERIFICATION_METHOD.SMS && (
-                            <VerificationPhoneInput loading={loading} onSendClick={handleSendSMSCode} />
-                        )}
-                    </Field>
-                </Row>
+                        </Field>
+                    </Row>
+                    <Row>
+                        <Label> </Label>
```

Comment:
It's gonna be angry about label not having content I think. Should maybe ask @nico3333fr about how to position this. I think we might need some class that adds a left margin equal to the label width...